### DISCUSSION TITLE
Upcoming Features ...

### DISCUSSION THREAD

**Discussion by BRAVO68WEB** [BRAVO68WEB/url-minify#133] [Ideas]
# GeoTracking

API :- https://ip.b68dev.xyz/

Example :- https://ip.b68dev.xyz/8.8.8.8

We are gonna store country and continent info in your API

# Chrome Extension

A simple chrome extension to shorten the URL of current tab on one click

**Comment by BRAVO68WEB** [comment]
@akankshat05 @Olaleye-Blessing wanna contribute?

**Comment by Olaleye-Blessing** [comment]
@BRAVO68WEB I've never created a chrome extension before but I think this one would be fun.

**Comment by BRAVO68WEB** [comment]
great

**Comment by Olaleye-Blessing** [comment]
so when should we start?

**Comment by akankshat05** [comment]
I would like to contribute

**Comment by BRAVO68WEB** [comment]
Currently I am building Backend Connectors to start from ...

**Comment by BRAVO68WEB** [comment]
@Olaleye-Blessing @akankshat05 Check `dev/ahead` branch

**Reply by BRAVO68WEB** [reply]
Can you make it more efficient

**Reply by BRAVO68WEB** [reply]
I mean the continent analytics part I used manual if else

**Comment by Olaleye-Blessing** [comment]
@BRAVO68WEB Sorry for the late reply. Which file did you use `if else`?

**Comment by Olaleye-Blessing** [comment]
And I thought we were going to create a new repository for creating chrome extension. Are you planning to use this same repository for the extension?

**Reply by BRAVO68WEB** [reply]
Yeah, actually it's a mono repo now

**Comment by Olaleye-Blessing** [comment]
ok, let me see what I can work on
